test/run.d

```diff
@@ -488,7 +488,7 @@ string[string] getEnvironment()
         auto druntimePath = environment.get("DRUNTIME_PATH", testPath(`..\..\druntime`));
         auto phobosPath = environment.get("PHOBOS_PATH", testPath(`..\..\phobos`));
         env["DFLAGS"] = `-I%s\import -I%s`.format(druntimePath, phobosPath);
-        env["LIB"] = phobosPath;
+        env["LIB"] = phobosPath ~ ";" ~ environment.get("LIB");
```

Comment:
Don't complete replace LIB, only preprend the dir containing fresh phobos.lib. We still need the original LIB dirs for kernel32.lib etc.